### ISSUE TITLE
Please add R language

### ISSUE THREAD

**Issue by c2lyw** [holzschu/a-shell#182]

**Comment by econcz** [comment]
I second this request

**Comment by holzschu** [comment]
Hi,
Thanks for the suggestion; I've added it to the list of things to add (but it's a long list). 
I'm curious, though: I thought R would make more sense inside Carnets (Notebooks app) than inside a-Shell (terminal app)?

**Comment by ifuchs** [comment]
I would agree that R in Carnets would be more useful than in a-shell. Of course, I am sure there are use cases for both. For that matter, I would also love to see a Julia kernel in Carnets.

**Comment by econcz** [comment]
> Hi,
> Thanks for the suggestion; I've added it to the list of things to add (but it's a long list).
> I'm curious, though: I thought R would make more sense inside Carnets (Notebooks app) than inside a-Shell (terminal app)?

R in Carnets would be even better.
